site/pages/v4/design/forms-fr.hbs

```diff
@@ -880,6 +881,66 @@
       </details>
     </div>
   </div>
+	<h4 id="fieldset"><span class="fa-stack"><span class="fa fa-circle fa-stack-2x"></span></span> Légende Tag Bordures</h4>
+  <p>Par défaut, toutes les balises <code>fieldset/legend</code> ont une bordure au-dessus d'elles pour séparer chaque fieldset si vous voulez que la bordure soit sous la classe <code>fieldset/legend</code> <code>.legend-brdr-bttm</code> sur la balise de <code>legend</code>.</p>
```

Comment:
@MaximMartel Can you review this please? Haven't looked at french since high-school.